apps/web/features/merch/services/api.ts

```diff
@@ -93,7 +93,7 @@ export class Api {
     return new Promise((res, rej) => {
       setTimeout(() => {
         res({
-          disabled: true,
+          disabled: false,
```

Comment:
This doesn't seem like an intended change.

Reply:
Noted, will revert